src/Microsoft.DotNet.Interactive/KernelSupportsNugetExtensions.cs

```diff
@@ -145,6 +145,11 @@ private static Command r()
             }
         }
 
+        private static bool EndsInDirectorySeparator(string path)
```

Comment:
This can be an extension method I suppose.

Reply:
is not used anywhere else

Reply:
It's a minor readability nitpick, not important.